### ISSUE TITLE
Overlap two events at the same from time in the full day view

### ISSUE THREAD

**Issue by ntung** [williamtroup/Calendar.js#245]
As you see, the View Timeline shows three while the View Full Day shows two.
**View Timeline**
![image](https://github.com/user-attachments/assets/59a1315e-3b80-4e55-8d09-e92cee67e5a7)
**View Full Day**: just two events because it overlaps two same from time events.
![image](https://github.com/user-attachments/assets/2c72fdb3-7519-4fc3-b31b-1e0421497fde)

**Comment by williamtroup** [comment]
Yes, the full-day view only supports two columns for events as it stands.  This will be improved in v3.0.0.

**Comment by ntung** [comment]
Do you have a plan to release v3.0.0?

**Comment by mesmaili151** [comment]
we have the same problem.. is there any update about the release?

**Comment by ntung** [comment]
@william-troup Do you plan to fix this issue? Thanks!
